content/applications/hr/recruitment/offer_job_positions.rst

```diff
@@ -0,0 +1,147 @@
+===================
+Offer job positions
+===================
+
+Once an applicant has successfully passed the various interview stages, the recruitment team is
+ready to send an offer for employment. The next step is to send the applicant a contract.
+
+.. seealso::
+   Refer to the :doc:`recruitment <../recruitment>` documentation for details on the various stages
+   of the recruitment process.
+
+Contract proposal
+=================
+
+When an offer is ready to be sent, first open the applicant's card by navigating to the
+:menuselection:`Recruitment app`, then click on the desired job position card.
+
+In the :guilabel:`Job Positions` Kanban view, the corresponding applicant card can be moved to the
+:guilabel:`Contract Proposal` stage. Drag and drop the applicant's card to the :guilabel:`Contract
+Proposal` stage, or click the :guilabel:`Contract Proposal` stage button in the top-right of the
+applicant's card.
+
+The next step is to send an offer to the applicant. On the applicant's card, click the
+:guilabel:`Generate Offer` button. A :guilabel:`Generate a Simulation Link` pop-up window appears.
+Most fields are pre-populated with information from the job position. If any necessary fields are
+blank, or if any information needs to be updated, enter or update the relevant information in the
+corresponding fields.
+
+.. note::
+   Depending on the localization setting for the company, and which applications are installed, some
+   fields may not appear in the :guilabel:`Generate a Simulation Link` pop-up window.
+
+   For example, if the *Fleet* application is **not** installed, any fields related to vehicles do
+   **not** appear.
+
+Universal fields
+----------------
+
+The following fields appear in the :guilabel:`Generate a Simulation Link` pop-up window, regardless
+of the localization.
+
+- :guilabel:`Contract Template`: the template currently being used to populate the
+  :guilabel:`Generate a Simulation Link` pop-up window. Use the drop-down menu to select a different
+  :guilabel:`Contract Template`, if desired.
+
+  .. note::
+     To modify the template, hover over the current template name, and click the :icon:`oi-launch`
+     :guilabel:`Internal link` icon that appears to the right of the drop-down arrow. Make any
+     desired changes, then click :guilabel:`Save & Close`.
+
+- :guilabel:`Job Position`: the name of the :guilabel:`Job Position` being offered to the applicant.
+  The selections available in the drop-down menu correspond to the job position configured on the
+  main *Recruitment* dashboard.
+- :guilabel:`Job Title`: the position being offered to the applicant.
```

Comment:
based on these definitions, it's a bit difficult to discern the differences between these two distinctions/fields...

Reply:
I honestly don't know why we have two fields for basically the same thing- except that one is not modifiable (has a drop down) while the other can be modified. So I am sure that's beneficial for some companies, somewhere. I'll add this info to these entires.

Reply:
OK i figured it out! I added an example admonition, where they are a Marketing Manager for a shoe company, but their scope is only the Children's line. So, they have 'Marketing Manager' selected from the drop-down Job Position field, BUT for the TITLE field, it's modified to say 'Marketing Manager: Children's Line'. So I think it's completely understandable now!